src/Sentry/Laravel/Profiling/FrameProcessor.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Sentry\Laravel\Profiling;
+
+use Illuminate\Support\Str;
+use Sentry\Laravel\Tracing\BacktraceHelper;
+
+class FrameProcessor
+{
+    /**
+     * @param array $frame
+     *
+     * @return array
+     */
+    public function __invoke(array $frame): array
+    {
+        // Check if we are dealing with a frame for a cached view path
+        if (Str::startsWith($frame['filename'], '/storage/framework/views/')) {
+            $originalViewPath = $this->backtraceHelper()->getOriginalViewPathForCompiledViewPath($frame['abs_path']);
```

Comment:
Could we benchmark the perf overhead?

Reply:
I think we could but not sure how to quantify, since this does a few I/O operations to read the view file. In this WIP I didn't implement memoizing which could reduce the amount of operations needed when a view is seen multiple times. I could run this code a few iterations on my machine and see what the result is, but it's always an increase since before we didn't do anything.

Reply:
A rough ballpark would suffice.